external/CMakeLists.txt

```diff
@@ -18,7 +18,8 @@ FetchContent_Declare(
   GIT_REPOSITORY https://github.com/google/googletest.git
   GIT_TAG v1.13.0
   # Patch for int to float conversion warning.
-  PATCH_COMMAND git cherry-pick -n 5cd81a7848203d3df6959c148eb91f1a4ff32c1d -m 1
+  PATCH_COMMAND git cherry-pick -n 5cd81a7848203d3df6959c148eb91f1a4ff32c1d -m 1 && git apply
+                "${CMAKE_CURRENT_SOURCE_DIR}/scoped_trace_terminate_on_thread_migrate.patch"
```

Comment:
It seems like this only works once. The `PATCH_COMMAND` seems to be run every time CMake is run, and the second time the `git apply` fails. I think we can `git reset` and re-`apply`, but I'm wondering if there are other options?

Reply:
The cherry-pick was always there and works fine when repeated. The apply fails the second time because the patch doesn't apply anymore.

Reply:
I now `git reset` every time before applying the patches. This seems to work, but is not particularly elegant.